pycbc/waveform/parameters.py

```diff
@@ -205,39 +205,39 @@ def docstr(self, prefix='', include_label=True):
 q = Parameter("q",
                 dtype=float, label=r"$q$",
                 description="The mass ratio, m1/m2, where m1 >= m2.")
-m_p = Parameter("m_p",
-                dtype=float, label=r"$m_{\mathrm{pr}}$",
+primary_mass = Parameter("primary_mass",
```

Comment:
@duncan-brown had requested that we change the labelling to be `m_1/2` to be consistent with plots produced by lalinference. Having `m_{pr}` had confused Stuart; he thought it meant source mass, which it didn't.